db/views/hmis_services_v03.sql

```diff
@@ -0,0 +1,44 @@
+-- To update, run rails generate scenic:view hmis_services --replace
```

Comment:
I want to flag one thing about services.

The view doesn't include all fields (`:other_type_provided, :moving_on_other_type, :sub_type_provided, :referral_outcome, :fa_amount, :fa_start_date, :fa_end_date`) that are resolved on the glq type [here](https://github.com/greenriver/hmis-warehouse/blob/75f2ca3677a5f32b5dbe0d2e2702f6c65c6d6244/drivers/hmis/app/graphql/types/hmis_schema/service.rb#L33-L35). I guess when those are resolved we [delegate](https://github.com/greenriver/hmis-warehouse/blob/75f2ca3677a5f32b5dbe0d2e2702f6c65c6d6244/drivers/hmis/app/models/hmis/hud/hmis_service.rb#L30-L38) to the owner record. Just want to call that out, it may have been very janky of me, perhaps it would be better to have all the fields on the view.

Reply:
That's good to call out, it looks like it could cause n+1 queries. But the issues we see around the service view are different and more severe. Maybe the n+1 queries could be addressed in another round of performance improvements